.semaphoreci/job1.sh

```diff
@@ -2,4 +2,3 @@
 set -e
 
 make
-if [ "$BRANCH_NAME" = "master" ]; then make push-docker ; fi
```

Comment:
No more pushing docker?

Reply:
For now on the master branch no. We will only push stable version. 
For now it will be easier to manage